prepare.sh

```diff
@@ -4,4 +4,44 @@ set -x
 
 yum -y install git podman
 
-git clone --single-branch --branch=master https://github.com/ceph/ceph-csi /opt/build
+while test $# -gt 0; do
+  case "$1" in
+    -h|--help)
+      echo " "
+      echo "options:"
+      echo "--help|-h                 specify the flags"
+      echo "--ref                     specify the reference of pr"
+      echo "--workdir                 specify the working directory"
+      exit 0
+      ;;
+
+#pr request will be passed through --ref flag
+    --ref*)  
+      export ref=`echo $1 | sed -e 's/^[^=]*=//g'`
```

Comment:
there is not need to `export` the variables, you can just use them without that insided this script.